### PR TITLE
Add docker invoke task

### PR DESCRIPTION
**Pull request by ogaca-dd**
What does this PR do?
---------------------

Add the invoke tasks `create-docker` and `destroy-docker`.

### CODE REVIEW

File: tasks/doc.py [hunk 1]
```diff
@@ -2,5 +2,6 @@
 
 install_agent: str = f"Install the Agent (default {tool.get_default_agent_install()})."
 agent_version: str = "The version of the Agent for example '7.42.0~rc.1-1' or '6.39.0 (default `latest`)'"
+agent_docker_version: str = "The docker version of the Agent for example '7.45.0-rc.3' (default `latest`)'"
```
Comment (pducolin) [inline tasks/doc.py:5]: **💬 suggestion**
We should support also agent full path option, useful to test pre-release, pre-rc versions

File: tasks/docker.py [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+
+    command = (
+        f'\nssh {user}@{host} "sudo usermod -aG docker {user} && sudo reboot"\n'
+        + f'docker context create pulumi-{host} --docker "host=ssh://{user}@{host}"\n'
+        + f'echo "Wait host to restart. If the next command fails, please wait and retry"; sleep 30\n'
+        + f"docker --context pulumi-{host} container ls\n"
+    )
+    pyperclip.copy(command)
+    print(
+        f"If you want to use docker context, you can run the following commands which were copied in the clipboard\n\n{command}"
```
Comment (pducolin) [inline tasks/docker.py:55]: **👏 praise**
Love this !

File: tasks/vm.py [hunk 3]
```diff
@@ -44,6 +45,20 @@ def create_vm(
...
+def _show_connection_message(full_stack_name: str):
+    outputs = tool.get_stack_json_outputs(full_stack_name)
+    connection = tool.Connection(outputs)
+    host = connection.host
+    user = connection.user
+
+    command = f"ssh {user}@{host}"
+    pyperclip.copy(command)
+    print(
+        f"\nYou can run the following command to connect to the host `{command}`. This command was copied to the clipboard\n"
```
Comment (pducolin) [inline tasks/vm.py:60]: **👏 praise**
Yessss

**Review by pducolin — APPROVED** [review]
One non blocker comment, love the usage of the clipboard

File: tasks/doc.py (hunk 1 above)
Comment (ogaca-dd) [reply tasks/doc.py:5]: Do you mean supporting the repository path?

Comment (ogaca-dd) [reply tasks/doc.py:5]: I think it can be added in a follow PR.

Comment (pducolin) [reply tasks/doc.py:5]: Rather the full path to the image, more friendly than using version + repo path, as you can copy and paste the full path from the CI output. Ok for the follow up !

**Pull request merged**